calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/camera_calibrator.py

```diff
@@ -787,6 +852,22 @@ def process_evaluation_results(
     def on_consumed(self):
         self.data_source.consumed()
 
+    def save_parameters(self, filename):
+        data_coll_params = self.data_collector.parameters_value()
+        board_params = self.board_parameters.parameters_value()
+        detector_params = self.detector.parameters_value()
+        calibrator_type = self.calibrator_type_combobox.currentData()
+        print(calibrator_type.value["name"], flush=True)
+        calib_params = self.calibrator_dict[calibrator_type].parameters_value()
+        with open(filename, "w") as file:
+            yaml.dump({"board_parameters": board_params}, file, default_flow_style=False)
```

Comment:
I think it is better to perform a single dump of a dictionary containing all the elements